invenio_app_ils/records/jsonresolver/circulation.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019 CERN.
+#
+# invenio-app-ils is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Ils Records Documents record circulation resolver."""
+
+import jsonresolver
+from werkzeug.routing import Rule
+
+from invenio_circulation.api import is_item_available, get_loan_for_item
+from invenio_circulation.search.api import search_by_pid
+
+from invenio_app_ils.circulation.utils import circulation_items_retriever
+from invenio_app_ils.records.api import Document
+
+
+@jsonresolver.hookimpl
+def jsonresolver_loader(url_map):
+    """Resolve the OpenAIRE grant."""
+    from flask import current_app
+
+    def _circulation_for_document_resolver(document_pid):
+        """Return circulation info for the given document."""
+        circulation = {}
+        active_loans = 0
+        loanable_items = 0
+        number_of_items = 0
+        items = circulation_items_retriever(document_pid)
+
+        for item in items:
+            number_of_items += 1
+            if not is_item_available(item):
+                active_loans += 1
+
+            if item["state"] is "LOANABLE":
```

Comment:
you can use `circulation_is_item_available` method (to be renamed in the future)